AreaCalculations/AreaCalculations/Calculations.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AreaCalculations
+{
+    class Calculations
+    {
+        Square square = new Square();
+        Rectangle rectangle = new Rectangle();
+        public enum Figures
+        {
+            Square = 1,
+            Rectangle
+        }
+        public Figures GetFigureType(string figure)
+        {
+            Figures result = 0;
+            switch (figure)
+            {
+                case "Квадрат":
+                    result = Figures.Square;
+                    break;
+                case "Прямоугольник":
+                    result = Figures.Rectangle;
+                    break;
+            }
+            return result;
+        }
+        public float GetResult(Figures figures)
```

Comment:
Please add an empty line before the method.
Also add descriptions before the class and description before each method.
(try to start with /// (three slashes before the method and see if it adds a template for you)

Reply:
Here are the examples of class description and methods description

![image](https://user-images.githubusercontent.com/22575418/105163291-c828cf00-5b24-11eb-93e9-0427a81690d2.png)